resources/analysisTools/roddyTools/wrapInScript.sh

```diff
@@ -330,14 +330,12 @@ else
   outputFileGroup=${outputFileGroup-$myGroup}
 
   exitCode=0
+  [[ ${disableDebugOptionsForToolscript-false} == true ]] && export WRAPPED_SCRIPT_DEBUG_OPTIONS=""
```

Comment:
Better change BashConverter.appendDebugVariables() (change it to produce `WRAPPED_SCRIPT_DEBUG_OPTIONS` and put it further upwards; see Roddy https://github.com/eilslabs/Roddy/pull/207)